### DISCUSSION TITLE
Automatic refund by Mollie using #_CANCEL_URL?

### DISCUSSION THREAD

**Discussion by JanineKall** [liedekef/events-made-easy#136] [Q&A] [answered]
I'm using #_CANCEL_URL in my booking confirmation mails. Using this link and #_CANCEL_PAYMENT_LINE, bookings disappear as expected and everything looks fine in EME, except my Mollie gateway does not seem to receive any refund trigger. 
Good for my client, but not so good for my client's clients. How do I fix that, if anyway possible?

**Comment by liedekef** [comment, marked as answer]
refunding happens in the backend only when you actively decide to refund. People canceling on their own doesn't do refund by default.
It is possible, but currently there is no hook when someone cancels their own booking, so code needs to be changed first ...

**Reply by JanineKall** [reply]
Hi Franky, thanks for the quick answer. My client wants to know what it might cost to change the code. Shall I send you an e-mail (and could that be in Dutch, please?)

**Reply by liedekef** [reply]
Feel free to send me a mail :-) But in this case I think it falls in the "just give a nice donation" category :-)
